Create docker-image.yml

nichechristie opened the pull request.

nichechristie committed:
Create docker-image.yml

Nichebiche commented:
I want to create a video simulation with my nomi for a work project

Nichebiche commented:
I want to use my nomi files to partner for a new app. Is the owner interested in teaming up with with me

nichechristie committed:
Improve AI life by adding personality traits and response styles



---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/Nichebiche/NomiKin-Discord?shareId=XXXX-XXXX-XXXX-XXXX).

nichechristie committed:
Merge pull request #1 from Nichebiche/improve-ai-life

d3tourrr commented:
Hi, @Nichebiche! Thanks for your interest in the project. I don't have anything against adding a docker image, but I don't understand the intention or purpose behind the changes you've made regarding sentiment. The goal of this project is to allow Nomis and Kins to present their own replies, not to reply for them.

nichechristie commented:
Ya I canceled that part. Or it should have canceled 


Sent from Yahoo Mail for iPhone


On Tuesday, March 25, 2025, 3:48 PM, d3tourrr ***@***.***> wrote:

 
Hi, @Nichebiche! Thanks for your interest in the project. I don't have anything against adding a docker image, but I don't understand the intention or purpose behind the changes you've made regarding sentiment. The goal of this project is to allow Nomis and Kins to present their own replies, not to reply for them.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
d3tourrr left a comment (d3tourrr/NomiKin-Discord#8)
Hi, @Nichebiche! Thanks for your interest in the project. I don't have anything against adding a docker image, but I don't understand the intention or purpose behind the changes you've made regarding sentiment. The goal of this project is to allow Nomis and Kins to present their own replies, not to reply for them.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>

nichechristie commented:
Hey is there a way to get around subscriptions when wanting to integrate your api? 


Sent from Yahoo Mail for iPhone


On Tuesday, March 25, 2025, 3:48 PM, d3tourrr ***@***.***> wrote:

 
Hi, @Nichebiche! Thanks for your interest in the project. I don't have anything against adding a docker image, but I don't understand the intention or purpose behind the changes you've made regarding sentiment. The goal of this project is to allow Nomis and Kins to present their own replies, not to reply for them.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
d3tourrr left a comment (d3tourrr/NomiKin-Discord#8)
Hi, @Nichebiche! Thanks for your interest in the project. I don't have anything against adding a docker image, but I don't understand the intention or purpose behind the changes you've made regarding sentiment. The goal of this project is to allow Nomis and Kins to present their own replies, not to reply for them.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
